src/engine/sync/synccontrol.cpp

```diff
@@ -353,7 +358,7 @@ void SyncControl::trackBeatsUpdated(mixxx::BeatsPointer pBeats) {
 }
 
 void SyncControl::slotControlPlay(double play) {
-    m_pEngineSync->notifyPlaying(this, play > 0.0);
+    m_pEngineSync->notifyPlayingAudible(this, play > 0.0 && m_audible);
```

Comment:
I don't know if this was caused by this PR (maybe it was just a fluke), but this segfaulted for me when running tests. When I ran the tests again, everything passed.

Here' the backtrace from the coredump:

```
Program terminated with signal SIGSEGV, Segmentation fault.
#0  0x000055e2095da8eb in SyncControl::slotControlPlay (this=0x55e20b5a0660, play=0) at /home/jan/Projects/mixxx/src/engine/sync/synccontrol.cpp:384
384	    m_pEngineSync->notifyPlayingAudible(this, play > 0.0 && m_audible);
[Current thread is 1 (Thread 0x7fb510b84640 (LWP 259384))]
>>> bt
#0  0x000055e2095da8eb in SyncControl::slotControlPlay(double) (this=0x55e20b5a0660, play=0) at /home/jan/Projects/mixxx/src/engine/sync/synccontrol.cpp:384
#1  0x00007fb524ce075b in  () at /usr/lib/libQt5Core.so.5
#2  0x000055e2094f5d40 in ControlProxy::valueChanged(double) (this=<optimized out>, _t1=<optimized out>) at /home/jan/Projects/mixxx/build/mixxx-lib_autogen/include/moc_controlproxy.cpp:164
#3  0x00007fb524ce075b in  () at /usr/lib/libQt5Core.so.5
#4  0x000055e2094ea625 in ControlDoublePrivate::valueChanged(double, QObject*) (this=this@entry=0x55e20b550a90, _t1=<optimized out>, _t1@entry=0, _t2=<optimized out>) at /home/jan/Projects/mixxx/build/mixxx-lib_autogen/include/moc_control.cpp:145
#5  0x000055e2094eadae in ControlDoublePrivate::setInner(double, QObject*) (this=0x55e20b550a90, value=0, pSender=<optimized out>) at /home/jan/Projects/mixxx/src/control/control.cpp:223
#6  0x00007fb524ce075b in  () at /usr/lib/libQt5Core.so.5
#7  0x000055e2094ea5b3 in ControlDoublePrivate::valueChangeRequest(double) (this=this@entry=0x55e20b550a90, _t1=<optimized out>) at /home/jan/Projects/mixxx/build/mixxx-lib_autogen/include/moc_control.cpp:152
#8  0x000055e2094eaee8 in ControlDoublePrivate::set(double, QObject*) (this=0x55e20b550a90, value=<optimized out>, pSender=0x55e20b5509f0) at /home/jan/Projects/mixxx/src/control/control.cpp:208
#9  0x000055e2095b0646 in ControlObject::set(double) (value=<optimized out>, this=<optimized out>) at /home/jan/Projects/mixxx/src/control/controlobject.h:83
#10 EngineBuffer::slotTrackLoading() (this=0x55e20b548a10) at /home/jan/Projects/mixxx/src/engine/enginebuffer.cpp:521
#11 0x00007fb524ce075b in  () at /usr/lib/libQt5Core.so.5
#12 0x00007fb524ce075b in  () at /usr/lib/libQt5Core.so.5
#13 0x000055e209584f20 in CachingReaderWorker::loadTrack(std::shared_ptr<Track> const&) (this=0x55e20b54bea8, pTrack=std::shared_ptr<Track> (use count 5, weak count 0) = {...}) at /home/jan/Projects/mixxx/src/engine/cachingreader/cachingreaderworker.cpp:136
#14 0x000055e209586f6d in CachingReaderWorker::run() (this=0x55e20b54bea8) at /home/jan/Projects/mixxx/src/engine/cachingreader/cachingreaderworker.cpp:104
#15 0x00007fb524abdfef in  () at /usr/lib/libQt5Core.so.5
#16 0x00007fb5245b7259 in start_thread () at /usr/lib/libpthread.so.0
#17 0x00007fb5244e05e3 in clone () at /usr/lib/libc.so.6
>>>
```